src/osd/PG.cc

```diff
@@ -7381,7 +7382,8 @@ boost::statechart::result PG::RecoveryState::Active::react(const MInfoRec& infoe
   // don't update history (yet) if we are active and primary; the replica
   // may be telling us they have activated (and committed) but we can't
   // share that until _everyone_ does the same.
-  if (pg->is_actingbackfill(infoevt.from)) {
+  if (pg->is_actingbackfill(infoevt.from) &&
+      pg->peer_activated.count(infoevt.from) == 0) {
```

Comment:
Is this from "osd/PG: ignore dup activation messages" an unrelated bug fix?

Reply:
yeah, unrelated fix for issue i encountered while testing this